examples/relay-angular-chat/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "relay-angular-chat",
```

```suggestion
  "name": "@waku/relay-angular-chat",
```
FYI I control the @waku orga on npmjs so feel free to always prepend so we don't have bug hunters reserving the name on npmjs and claiming a bounty (I know you have set the `private: true`).